acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -12,17 +12,13 @@
 test_name 'Default auth.conf rules'
 
 step 'Turn on new auth support' do
-  modify_tk_config(
-    master,
-    File.join(master['puppetserver-confdir'], 'puppetserver.conf'),
-    {'jruby-puppet' => {'use-legacy-auth-conf' => false}})
+  modify_tk_config(master, master['puppetserver-config'],
```

Comment:
I guess the other code is referencing these through `options['puppetserver-config']`...this works but I just noticed that, should probably change it to be consistent?